phase2/addon/lib/presentation/doorhanger.js

```diff
@@ -590,6 +596,14 @@ function stop(){
   }
 }
 
+function buttonInited(){
+  return !!button;
+}
+
+function buttonPlacement(){
+  return require('../utils').findButtonPlacement("toggle-button--guide-cfr-1shieldmozillaorg-dh-button");
```

Comment:
the button id looks weird, could possibly be generated from the addon id (if "@" s are removed).